dune-project

```diff
@@ -30,4 +30,6 @@
   ppx_enumerate
   ppx_fields_conv
   (ppx_yojson_conv (<= v0.15.1))
-  ptime))
+  ptime
+  (ocamlformat :with-dev-setup)
+  (ocp-indent :with-dev-setup)))
```

Comment:
Why do you want `ocp-indent` as a dev dependency?

About `ocamlformat`: If we add it as a dependency, we need to specify its version. In that case, in the future, we'd need to update both the ocamlformat and the dune-project file when bumping `ocamlformat`. I don't have a strong opinion about this. If you want to add `ocamlformat` with version as dev dependency, that's fine for me.

Reply:
I'm curious: Why do you still use ocp-indent while also using ocamlformat?

In any case, you can rely on ocp-indent! I'm just wondering why you'd impose it on others by making it a dev dependency.